packages/odata-v2/src/request-builder/batch-request-builder.spec.ts

```diff
@@ -70,6 +87,51 @@ ${header}
         response[0].body
       );
       expect(casted).toEqual('MyText');
+    } else {
+      throw new Error('Should be readResponse');
+    }
+  });
+
+  it('batch works with action imports', async () => {
```

Comment:
```suggestion
  it('batch works with POST function imports', async () => {
```

Or something similar for clarity, since action imports are only in v4